app/src/org/commcare/AppUtils.java

```diff
@@ -173,9 +173,7 @@ public static String getCurrentVersionString() {
         String buildNumber = BuildConfig.BUILD_NUMBER;
 
         return Localization.get(application.getString(R.string.app_version_string), new String[]{
-                BuildConfig.VERSION_NAME,
-                String.valueOf(BuildConfig.VERSION_CODE),
-                ccv, buildNumber, buildDate, profileVersion});
+                ccv, String.valueOf(BuildConfig.VERSION_CODE), ccv, buildNumber, buildDate, profileVersion});
```

Comment:
can we not show `ccv` twice ?

Reply:
Yes, [7f0f857](https://github.com/dimagi/commcare-android/pull/2978/commits/7f0f857cc7f56a4710438ed6d76ceacd21f9b342)

Reply:
I wonder if we still need this notation, having it twice when the source is the same seems redundant